### DISCUSSION TITLE
Add Projects to the mobile app

### DISCUSSION THREAD

**Discussion by safeboot** [community/community#8459] [Mobile] [answered]
Hi all,

I started using the GitHub mobile app and found that there is no Projects feature like on desktop.

It would be incredibly handy to have it on mobile to track my To-dos and add more ideas as I see fit.

Would this be possible?

**Comment by mxie** [comment, marked as answer]
Hi @safeboot - thanks for sharing your feedback! We're currently in the process of looking into Projects support. Please check out this [existing discussion](https://github.com/github/feedback/discussions/3023) to provide additional input as well as track future updates from us on this feature.
